### DISCUSSION TITLE
How to disable annoying Info and warnings on pylint

### DISCUSSION THREAD

**Discussion by anomaloussus** [nvimtools/none-ls.nvim#94] [Q&A]
```local null_ls = require("null-ls")
null_ls.setup({
    sources = {
        null_ls.builtins.diagnostics.cppcheck,
        null_ls.builtins.diagnostics.pylint,
        null_ls.builtins.completion.spell,
    },
})
```
I want it to only show errors not stuff like you are missing a docsrting, tried this but doesn't  work   ```null_ls.builtins.diagnostics.pylint.with({
            args = { "--disable=R,C" } 
        }), ```

**Comment by sblask** [comment]
There is probably a way to do that with none-ls, but you might be better off with a pylint config file?
